spec/models/outgoing_message_spec.rb

```diff
@@ -68,7 +68,7 @@
 
   describe '.expected_send_errors' do
     subject { described_class.expected_send_errors }
-    class TestError < StandardError ; end
+    class TestError < StandardError; end
```

Comment:
Still has a semi colon! When defining empty classes I prefer:

```suggestion
    TestError = Class.new(StandardError)
```

Reply:
This cop is just checking that there's no _space_ before the semicolon, rather than the semicolon not existing at all.

Not sure about `TestError = Class.new(StandardError)` – but that's a question for another cop anyway.